python/arcticdb/vector_db/vector_db.py

```diff
@@ -0,0 +1,521 @@
+import warnings
+import pandas as pd
+import numpy as np
+
+from collections import namedtuple
+from typing import Union, Optional, Collection, List, Mapping
+
+from arcticdb import Arctic
+from arcticdb.options import LibraryOptions
+from arcticdb.version_store.library import Library, ArcticUnsupportedDataTypeException
+from arcticdb.version_store.processing import QueryBuilder
+from arcticdb_ext.version_store import TopKClause as _TopKClause, NoSuchVersionException
```

Comment:
The clause should be an unused import now

Reply:
Yep